web/src/app/pages/create-survey/survey-loi/survey-loi.component.ts

```diff
@@ -43,7 +43,11 @@ export class SurveyLoiComponent {
   ) {}
 
   async ngOnInit() {
-    this.lois$ = this.loiService.getLocationsOfInterest$();
+    this.loiService
+      .getLocationsOfInterest$()
```

Comment:
Can we reuse the logic in EditJob, perhaps refactor it into the loiService?

I.e.:
```
function getPredefinedLoisByJobId(jobId) {
}
```

Reply:
yes

Reply:
Great. Will you do it?